package.json

```diff
@@ -80,7 +86,8 @@
     "test:build": "mocha ./tests/test-build.js",
     "test:headless": "node ./tests/headless.js",
     "test:karma": "npm run karma",
-    "prepare": "npm run snyk-protect && npm run build:components"
+    "prepare": "npm run snyk-protect && npm run build:components",
+    "test": "jest"
```

Comment:
Probably want “test:jest” here, until we fully remove Karma.

Reply:
Or we can just `npx jest`.